readme.md

```diff
@@ -170,6 +170,24 @@ import { config } from "favicons";
 
 For the full list of files, check `config/files.json`. For the full HTML code, check `config/html.js`. Finally, for the full list of icons, check `config/icons.json`.
 
+To save the files that are output, you can use `fs`.
+
+For example, amend the code above for Node.js as follows:
+
+```
+var fs = require ("fs");
+
+try {
+  const response = await favicons(source, configuration);
+  response.images.map (image => {
```

Comment:
Example is not full. There could be also `files` and `html` fragments.

Reply:
I'm not sure how you want to handle this.  Feel free to change the code as you see fit.

Reply:
@Elysiome Thanks! I will close this PR in favor of yours.